front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -322,12 +322,12 @@ async function handler(
 
       if (dataSource.vault.isSystem()) {
         const globalVault = await VaultResource.fetchWorkspaceGlobalVault(auth);
-
-        await DataSourceViewResource.createViewInVaultFromDataSourceIncludingAllDocuments(
+        // TODO
```

Comment:
don't we need to implement this todo now ?

Reply:
Oups, leftover from personal reminder